doc/site/en/guides/border-router/beaglebone-black.md

```diff
@@ -1,338 +1,204 @@
-The partition table has been altered.
-Calling ioctl() to re-read partition table.
-Re-reading the partition table failed.: Device or resource busy
-
-The kernel still uses the old table. The new table will be used at the next
-reboot or after you run partprobe(8) or kpartx(8).
+sudo shutdown -r now
 ```
 
-### 3. Resize the file system
+This will close your SSH session.
 
-Use `resize2fs` to resize the image file system to the newly-expanded partition
-size. This tool expands or shrinks a file system.
-
-Reboot the BBB.
-
-Some images may throw an `fsck` error upon reboot. `fsck` runs
-automatically on boot and checks for file system consistency. If you get
-this error, ignore it and wait about 20 seconds for the login
-prompt to appear.
+Once logged back into the BeagleBone install `network-manager` with the command
+`sudo apt-get install network-manager`. Then disable `connman` and enable
+`network-manager`:
 
 ```
-Loading, please wait...
-[    4.873285]  remoteproc1: failed to load am335x-pru0-fw
-[    4.918852]  remoteproc1: request_firmware failed: -2
-[    4.924046] pru-rproc 4a334000.pru0: rproc_boot failed
-[    5.052414]  remoteproc1: failed to load am335x-pru1-fw
-[    5.069652]  remoteproc1: request_firmware failed: -2
-[    5.074889] pru-rproc 4a338000.pru1: rproc_boot failed
-fsck: error 2 (No such file or directory) while executing fsck.ext4 for /dev/mmcblk0p1
-fsck exited with status code 8
+sudo systemctl disable connman
+sudo systemctl enable netowrk-manager
 ```
 
-Log in as `root`:
+If we were to `stop` connman directly here it would break the SSH session
+because the network interface is managed by connman. Instead we setup the
+system to take effect on the next boot. Now reboot the Beaglebone and re-login.
 
 ```
-$ sudo bash
+sudo shutdown -r now
 ```
 
-Resize the file system for the target partition:
+Network Manager may not have setup the DNS name servers. Edit `resolv.conf`
+with the command `sudo vim /etc/resolv.conf` and make sure the contents contain
+the Google DNS and Cloudflare DNS:
 
 ```
-root@beaglebone:/home/debian# resize2fs /dev/mmcblk0p1
-resize2fs 1.43 (17-May-2016)
-Filesystem at /dev/mmcblk0p1 is mounted on /; on-line resizing required
-old_desc_blocks = 1, new_desc_blocks = 1
-The filesystem on /dev/mmcblk0p1 is now 1835008 (4k) blocks long.
+nameserver 8.8.8.8
+nameserver 1.1.1.1
 ```
 
-Reboot the BBB. If you encountered the `fsck` issue, rebuild the `initramfs`, which is
-the initial RAM file system used when Linux boots.
+Restart to make sure Network Manager is setup correctly.
 
 ```
-$ sudo update-initramfs -u
-update-initramfs: Generating /boot/initrd.img-4.4.54-ti-r93
+sudo shutdown -r now
 ```
 
-Reboot the BBB again. It should boot without the `fsck` error.
-
-## Step 4: Build and install OTBR
+Note: If your BeagleBone has a WiLink based Wi-Fi module installed, the
+following steps may be applicable to you. This was tested with a
+BBONE-GATEWAY-CAPE. Some of these may not be required.
```

Comment:
Callouts should be blockquote with > preceding the first line and the others indented

> Note: